docs/api/index.md

```diff
@@ -16,6 +16,7 @@ interface MountingOptions<Props, Data = {}> {
   attrs?: Record<string, unknown>
   data?: () => {} extends Data ? any : Data extends object ? Partial<Data> : any
   props?: (RawProps & Props) | ({} extends Props ? null : never)
+  vModel?: (RawProps & Props) | ({} extends Props ? null : never)
```

Comment:
In terms of typing, should we just extract the props that the user has defined? eg: `update:${Prop}`

This is because vue-3 already gives a warning when using an non declared event.